[bitnami/wordpress] Document MaintenanceMode limitation & add missing parameters (#3250)
bitnami/charts: bitnami/wordpress/README.md

+ metrics.enabled: true
```

Note that [values-production.yaml](values-production.yaml) includes a replicaCount of 3, so there will be 3 WordPress pods. As a result, to use the "/admin" portal and to ensure you can scale wordpress you need to provide a ReadWriteMany PVC, if you don't have a provisioner for this type of storage, we recommend that you install the [NFS Server Provisioner chart](https://github.com/helm/charts/tree/master/stable/nfs-server-provisioner) (with the correct parameters, such as `persistence.enabled=true` and `persistence.size=10Gi`) and map it to a RWO volume.

Then, you can deploy WordPress chart using the proper parameters:

```console
persistence.storageClass=nfs
mariadb.master.persistence.storageClass=nfs
```

#### Known limitations

> When performing admin operations that require activating the maintenance mode (such as updating a plugin or theme), it's activated in only one replica (this is related to some limitation that doesn't allow us to change the maintenance file path to include it in the NFS volume, find more information on [this ticket](https://core.trac.wordpress.org/ticket/50797)). This implies that WP could be attending requests on other replicas while performing admin operations, with unpredictable consequences.

Reviewer: I don't think we need to explain the background here, this should be fine:

```suggestion
When performing admin operations that require activating the maintenance mode (such as updating a plugin or theme), it's activated in only one replica (see: [bug report](https://core.trac.wordpress.org/ticket/50797)). This implies that WP could be attending requests on other replicas while performing admin operations, with unpredictable consequences.
```

> To avoid that, you can manually activate/deactivate the maintenance mode on every replica using the WP CLI. For instance, if you installed WP with replicas you can run the commands below to activate the maintenance mode (the following example assumes that the release name is `wordpress`):

Reviewer: ```suggestion
To avoid that, you can manually activate/deactivate the maintenance mode on every replica using the WP CLI. For instance, if you installed WP with three replicas, you can run the commands below to activate the maintenance mode in all of them (assuming that the release name is `wordpress`):
```
